code-challenges.test.js

```diff
@@ -45,12 +107,42 @@ const letterE = "e"
 
 // a) Create a test with expect statements using the variable provided.
 
+// I really didnt get this one. I wanted to iterate through the array and setting up a if statement matching the two first values and checking if they are the same
+// use the the .map function 
+// return a boolean vale 
+
+describe("addUp", () => {
+
+    const hand1 = [5, 5, 5, 3, 3]
+    const hand2 = [5, 5, 3, 3, 4]
+    const hand3 = [5, 5, 5, 5, 4]
+
+ 
+    it("Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind..", () => {
+  
+      
+      expect(addUp(hand1)).toEqual(true)
+      expect(addUp(hand2)).toEqual(false)
+      expect(addUp(hand3)).toEqual(false)
+     
+    })
+  })
+
+
 const hand1 = [5, 5, 5, 3, 3]
 // Expected output: true
 const hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+const addUp = (array) => {
+    for (array.map){
+        if i==i 
+    }
+    return array
+}
```

Comment:
This is a tough problem for sure. I would start with double checking the iteration approach. It looks like you are using both a for loop and a map.